app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,69 @@
+class DriversController < ApplicationController
+  def index
+    @drivers = Driver.all
+  end
+
+  def new
+    @driver = Driver.new
+  end
+
+  def create
+    @driver = Driver.new(driver_params)
+
+    if @driver.save
+      redirect_to driver_path(@driver.id)
+    else
+    render :new
+    end
+  end
+
+  def show
+    driver_id = params[:id].to_i
+    @driver = Driver.find_by(id: driver_id)
+    #
```

Comment:
Nitpicking: This is an empty line with a comment, but no comment :) feel free to delete this before submission